lib/socket/socket_c.nit

```diff
@@ -482,3 +480,52 @@ extern class NativeSocketPollValues `{ int `}
 		return self | other;
 	`}
 end
+
+redef class Sys
+	# Get network host entry
+	fun gethostbyname(name: NativeString): NativeSocketHostent `{
+		return gethostbyname(name);
+	`}
+
+	# Last error raised by `gethostbyname`
+	fun h_errno: HErrno `{ return h_errno; `}
```

Comment:
It's the exact name of the C global variable. As per our nity/native FFI modules pattern, in the native layer we use the native names, so that experienced C users can recognised those services. The only problem here is that the native modules should be private. Which is a TODO for the socket module.